pkg/apis/upload/v1alpha1/openapi_generated.go

```diff
@@ -23,149 +23,9 @@ limitations under the License.
 package v1alpha1
 
 import (
-	spec "github.com/go-openapi/spec"
 	common "k8s.io/kube-openapi/pkg/common"
 )
 
 func GetOpenAPIDefinitions(ref common.ReferenceCallback) map[string]common.OpenAPIDefinition {
-	return map[string]common.OpenAPIDefinition{
```

Comment:
Appears that you did not use `make generate' to do code generation.

Reply:
Can we add a Jenkins validation that can detect this automatically?

Reply:
@aglitke we already have that.  This build didn't get that far.  Travis build appears to have got wedged sometime before that.

Reply:
Travis is currently having issues starting linux VMs, if you look at their status page you will see they are investigating. Also STDCI is not working right, I have engaged the maintainers to see what is going on.